src/NServiceBus.Persistence.DynamoDB/Saga/SagaPersister.cs

```diff
@@ -15,11 +15,13 @@ class SagaPersister : ISagaPersister
     {
         readonly SagaPersistenceConfiguration configuration;
         readonly IAmazonDynamoDB dynamoDbClient;
+        readonly DynamoDBContext dynamoDbContext;
 
         public SagaPersister(SagaPersistenceConfiguration configuration, IAmazonDynamoDB dynamoDbClient)
         {
             this.configuration = configuration;
             this.dynamoDbClient = dynamoDbClient;
+            dynamoDbContext = new DynamoDBContext(this.dynamoDbClient);
```

Comment:
since we assume the operations should bypass all validations, could we pass a fake/null client instead?